### PR TITLE
Update toggle, checkbox, and radio components

### PR DESCRIPTION
**Pull request by ellynliu**
Task: task-988025

Link: [preview](http://localhost:1111/)

Update toggle, checkbox, and radio to match fluent 2 styles

## Testing

1. http://localhost:1111/components/toggle.html
2. http://localhost:1111/components/checkbox.html
3. http://localhost:1111/components/radio.html

## Additional information

[Optional]

## Contributor checklist

- [ ] Did you update the description of this pull request with a review link and test steps?
- [ ] Did you submit a changeset? Changesets are required for all code changes.
- [ ] Does your pull request implement complex UI logic (js/ts)? Consider adding an integration test to test your user flow.
- [ ] Does your pull request add a new page to the documentation site? Add your new page for automated accessibility testing in /integration/tests/accessibility.

### CODE REVIEW

File: .changeset/curvy-numbers-sing.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Update toggle and checkbox to match fluent 2
```
Comment (ollips) [inline .changeset/curvy-numbers-sing.md:5]: I would avoid mentioning fluent un changeset. What about this:
`Updating icons in toggle and checkbox components.`

File: css/src/components/toggle.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 $toggle-transition: $input-transition-duration cubic-bezier(0.01, 1.23, 0.58, 0.96) !default;
-$toggle-circle-size: 0.75em !default;
-$toggle-circle-top: 0.1875em !default;
+$toggle-circle-size: 0.85em !default;
```
Comment (ollips) [inline css/src/components/toggle.scss:2]: similar should be done for `radio` component:

https://design.learn.microsoft.com/components/radio.html

image

vs Atlas:

image

Comment (ellynliu) [reply css/src/components/toggle.scss:2]: updated

File: .changeset/curvy-numbers-sing.md (hunk 1 above)
Comment (ellynliu) [reply .changeset/curvy-numbers-sing.md]: updated

File: css/src/components/form/radio.scss [hunk 3]
```diff
@@ -23,7 +23,7 @@ $radio-spacing: 0.5em !default;
 
 @mixin radio-dot-checked {
 	background: $radio-dot-checked-color;
-	box-shadow: inset 0 0 0 0.25em $body-background;
+	box-shadow: inset 0 0 0 0.15em $body-background;
```
Comment (homing1) [inline css/src/components/form/radio.scss:26]: It looks like hovered check on line 97 should be updated as well

image

Comment (ellynliu) [reply css/src/components/form/radio.scss:26]: updated!

**Review by ollips — APPROVED** [review]

File: css/src/components/form/radio.scss (hunk 3 above)
Comment (homing1) [reply css/src/components/form/radio.scss:26]: Should we make a new variable for the 0.15em so it can be shared with the hovered check?

Comment (ellynliu) [reply css/src/components/form/radio.scss]: pushed

**Review by homing1 — APPROVED** [review]

**Review by ollips — APPROVED** [review]

**Pull request merged**